api/src/main/java/io/minio/MinioClient.java

```diff
@@ -825,7 +825,7 @@ private Request createRequest(Method method, String bucketName, String objectNam
     }
 
     HttpUrl.Builder urlBuilder = this.baseUrl.newBuilder();
-
+    
```

Comment:
Any particular reason for adding this empty line?

Reply:
@aead Made the changes.